package.json

```
@@ -3,10 +3,22 @@
   "description": "An Esprima-compatible JavaScript parser built on Acorn",
   "author": "Nicholas C. Zakas <nicholas+npm@nczconsulting.com>",
   "homepage": "https://github.com/eslint/espree",
-  "main": "espree.js",
+  "main": "dist/espree.cjs",
+  "type": "module",
+  "exports": {
+    ".": [
+      {
+        "import": "./espree.js",
+        "default": "./dist/espree.cjs"
+      },
```

```suggestion
      {
        "import": "./espree.js",
        "require": "./dist/espree.cjs",        
        "default": "./dist/espree.cjs"
      },
```

I'm not sure if it makes any difference, but a similar change in acorn had [`"requre"`](https://github.com/acornjs/acorn/blob/74b59384320ced82e09da2e8fdbed16810f7379a/acorn/package.json#L12) too.